modules/validation/src/validation.ts

```diff
@@ -187,8 +194,8 @@ export const isInteger = (value: number): boolean => {
 /**
  * This can be used to validate resource names. For example
  * IdP names and Application names. This allows values like
- * { abc123, ab-c123, some-name, some_name, SOME-NAME } but
- * not values starting with { - _ numbers } only alphabet
+ * \{ abc123, ab-c123, some-name, some_name, SOME-NAME \} but
```

Comment:
```suggestion
 * ```{ abc123, ab-c123, some-name, some_name, SOME-NAME }``` but
```
Instead of escaping characters inside comments, we can wrap the example code, objects etc with triple backticks to keep eslint from throwing warnings over those.  

Eg: https://github.com/pavinduLakshan/identity-apps/blob/338a95bc63c94018475f9de9a096d7c6e0720bcd/modules/react-components/src/components/alert/confirmationModal-alert.tsx#L43

Reply:
Addressed with 52d4cc7d04defb5ba716ff35dcf56f35a5b9db4a
Didn't use triple backticks because it renders the block as a code block. Instead used single backtick.